spec/linter-stylelint-spec.js

```diff
@@ -4,8 +4,11 @@ import * as path from 'path';
 
 const badDir = path.join(__dirname, 'fixtures', 'bad');
 const configStandardPath = path.join(badDir, 'stylelint-config-standard.css');
+const badLessDir = path.join(__dirname, 'fixtures', 'less');
+const configStandardLessPath = path.join(badLessDir, 'stylelint-config-standard.less');
 const warn = path.join(__dirname, 'fixtures', 'warn', 'warn.css');
 const good = path.join(__dirname, 'fixtures', 'good', 'good.css');
+const goodLess = path.join(__dirname, 'fixtures', 'less', 'good.less');
```

Comment:
Use the directory variable you specified above, no reason to have two different specifications for the same folder.